llvm/test/CodeGen/RISCV/GlobalISel/rv64zbb.ll

```diff
@@ -794,6 +792,8 @@ define signext i32 @ctpop_i32_load(ptr %p) nounwind {
 ; RV64ZBB-LABEL: ctpop_i32_load:
 ; RV64ZBB:       # %bb.0:
 ; RV64ZBB-NEXT:    lw a0, 0(a0)
+; RV64ZBB-NEXT:    slli a0, a0, 32
```

Comment:
Is this needed or was this incorrect previously?

Reply:
I think this is a fallout from making i32 illegal

Reply:
@topperc are you planning on working on the load+zext combine too?

Reply:
> @topperc are you planning on working on the load+zext combine too?

Yes. It's already part of #115236. I forgot it showed up here too so I'll probably commit that directly after this patch.